src/data/actions/reviewMetaDataAction.jsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-undef */
+import axios from 'axios';
+import { GET_REVIEW_META_DATA_SUCCESS } from '../types/types';
+
+export const getReviewMetaData = () => (dispatch) => axios.get('http://3.134.102.30/reviews/1/meta')
+  .then((res) => dispatch(getReviewMetaDataSuccess(res.data)))
```

Comment:
is it possible to refactor the API call to utilize the shared api file? Is there a good reason to separate the API calls to multiple files?